test/system_test.py

```diff
@@ -355,6 +355,13 @@ def test_install_latest(self):
         spack_install_and_test('flexpart-ifs @meteoswiss-10')
 
 
+@pytest.mark.no_tsa  # flexpart-cosmo does not run on Tsa
```

Comment:
May I suggest the following wording then?
```suggestion
@pytest.mark.no_tsa  # No one uses flexpart-cosmo on Tsa
```
I misunderstood the old wording as: "flexpart-cosmo has a runtime problem on Tsa"

Reply:
May be more precise would be: No one uses spack for flexpart-cosmo on Tsa
@dominichofer we once tried this but it revealed to be too cumbersome to set up and thus not worth the effort for a machine already tagged to be phased-out .

Reply:
Yes, that's even better. I like it.